vendor/engines/c2po/lib/c2po.rb

```diff
@@ -9,17 +10,21 @@ class Engine < Rails::Engine
 
       # make this engine's views override the main app's views
       paths = ActionController::Base.view_paths.to_a
-      index = paths.find_index { |p| p.to_s.include? 'c2po' }
+      index = paths.find_index { |p| p.to_s.include? "c2po" }
       paths.unshift paths.delete_at(index)
       ActionController::Base.view_paths = paths
+
+      FacilityManagement.register_payment_type(CreditCardAccount, short_name: "cc")
+      FacilityManagement.register_payment_type(PurchaseOrderAccount, short_name: "po")
```

Comment:
for shortname i'd prefer `credit_card` and `purchase_order`, the few extra bytes are worth the improved readability